controllers/managedocs_controller.go

```diff
@@ -913,6 +977,42 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 	return nil
 }
 
+func getPrometheusWebSpec() *promv1.WebSpec {
```

Comment:
- just curious why can't the function body be stored in templates/prometheus.go?
- is it due to cross pkg vars?

Reply:
Yes, you are right. We are referring to the names of  package `controllers` and don't want to have a cyclic reference in the `templates` package by adding this function there.